pkg_config.10.xml

```diff
@@ -184,7 +184,7 @@
 		<build_pbi>
 			<ports_before>security/openssl</ports_before>
 			<custom_name>haproxy-devel</custom_name>
-			<port>net/haproxy</port>
+			<port>net/haproxy-devel</port>
```

Comment:
I hope `<run_depends>sbin/haproxy:net/haproxy-devel</run_depends>` is ok? I have commited that below in beaf604.